json_serializable/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 5.0.0-dev
```

Comment:
You might be able to consider not doing a breaking release here if this package is not intended to be used as a library for any reason?

Reply:
It can be used as a lib, but it's not widely supported.

I'll bump this back in a follow-up. I have other fixes I want to get in.